Api/Entities/StickupCamSettings.cs

```diff
@@ -1,20 +1,21 @@
-﻿using System.Text.Json.Serialization;
+﻿using System.Collections.Generic;
+using System.Text.Json.Serialization;
 
 namespace KoenZomers.Ring.Api.Entities
 {
     public class StickupCamSettings
     {
         [JsonPropertyName("enable_vod")]
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
-        public bool? EnableVod { get; set; }
+        public long? EnableVod { get; set; }
 
         [JsonPropertyName("exposure_control")]
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
         public long? ExposureControl { get; set; }
 
         [JsonPropertyName("motion_zones")]
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
-        public MotionZone MotionZones { get; set; }
+        public List<MotionZone> MotionZones { get; set; }
```

Comment:
As mentioned on your issue, on my end, I'm seeing the `stickup_cams.settings.motion_zones` array come back as an array of five integers (and in my case, the integers are always either 0 or 1).  I don't think those will deserialize to `List<MotionZone>`.  Have I misunderstood?

Reply:
Correct, they won't. I'm only having a Ring Doorbell myself and as the API is not documented nor supported to be called, its tricky for me to code this so it will always work. Let me see if there's a way it can be ignored if it doesn't parse somehow.

Reply:
Seems System.Text.Json has no such way (yet) and I don't want to jump back to Newtonsoft for it. For me it doesn't return anything as I don't have those devices. I will just take this section out for now. Hopefully that works for everyone.